keycloak/keycloak-admin-ui
# Improved Learn More links

## Motivation
Improved Learn More links as per #2571

## Verification Steps
1. Go to Realm roles and select `Learn More` - This will work once https://github.com/keycloak/keycloak-documentation/pull/1517 is merged
2. Go to Groups and select `Learn More`.
3. Verify that the correct URL is displaying.

## Checklist:
